baseapp_ratings/graphql/object_types.py

```
@@ -71,7 +71,7 @@ class Meta:
         )
 
     @classmethod
-    def get_node(self, info, id):
+    def get_node(cls, info, id):
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)